### ISSUE TITLE
Wish you publish the new version of the turbogears2 framework

### ISSUE THREAD

**Issue by ockan** [TurboGears/tg2#140]
Hi,Amol-,

the framework did not update publishing for 4 years,
but we saw you still update the files in GitHub.

would you publish the new version of the turbogears2 framework?

**Comment by amol-** [comment]
Yes, I have been working on 2.5, but given how much has changed in the Python ecosystem in the last years it's taking a lot of time to refactor things. The core framework itself is mostly done for 2.5, but there is a lot of work pending in upgrading the full stack based on it.

**Comment by ockan** [comment]
Good news.you are great, we look forward to the next coming version 2.5

**Comment by amol-** [comment]
So far in the TurboGears ecosystem the following packages have been released with Python 3.13 support:

- [Tempita 0.6](https://github.com/TurboGears/tempita/releases/tag/0.6)
- [Gearbox 0.3](https://github.com/TurboGears/gearbox/releases/tag/0.3.0)

**Comment by amol-** [comment]
TurboGears 2.5.0 has been released, see [upgrade guidelines ](https://turbogears.readthedocs.io/en/latest/reference/upgrading.html#from-2-4-3-to-2-5-0)

* [TurboGears 2.5.0](https://github.com/TurboGears/tg2/releases/tag/tg2.5.0)
* [tg2.devtools 2.5.0](https://github.com/TurboGears/tg2devtools/releases/tag/tg2.5.0)

Note that the documentation might still refer to parts that are no longer valid in 2.5.0, if you find any please open a ticket so that the documentation can be upgraded.
